oci-library/oci-hol/java-management/set-up-logrotate/set-up-logrotate.md

```diff
@@ -0,0 +1,138 @@
+# Set up logrotate
```

Comment:
"Set up logrotate" to be changed to "Set Up Logrotate" as per Self QA instructions section 3 (Section 3: Workshop Title & Lab Title)

Reply:
Self QA mentioned sentence casing, not title casing. Can you double check again?

Reply:
![image](https://user-images.githubusercontent.com/101088913/159838313-61a4b574-1951-490f-b2a4-bd7bf89f3c03.png)

Agree the recommended way is to use sentence casing. But I think at least 'logrotate' should be title case as it is some type of product (software product).

also current JMS Livelabs Lab 1 has 'Set Up' in title case https://apexapps.oracle.com/pls/apex/dbpm/r/livelabs/workshop-attendee-2?p210_workshop_id=912&p210_type=2&session=14618932106869

Reply:
ok i see the problem. the previous QA instruction did not specifically mention sentence case, so the initial labs were all using Title case. I will make those adjustments to earlier labs

Agree with capitalizing "logrotate" since its a product. 

Btw capitalization does not equate to title casing, title case involves capitalizing all major words